app/api/data.py

```
@@ -0,0 +1,1157 @@
+    week: Optional[int] = None,
+    season: Optional[int] = None,
+    team_id: Optional[str] = None,
+    status: Optional[str] = None,
+    game_type: Optional[str] = Query(
+        None, description="Game type: regular, playoff, championship"
+    ),
+    primetime: Optional[bool] = Query(None, description="Filter primetime games"),
+    include_predictions: bool = Query(False, description="Include game predictions"),
+    include_sentiment: bool = Query(False, description="Include game sentiment data"),
+    limit: int = Query(20, ge=1, le=100, description="Number of games to return"),
+    db=Depends(get_database),
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)